InterviewService.swift

```diff
@@ -0,0 +1,71 @@
+
+import Foundation
+
+enum Mode {
+    case preprod
+    case prod
```

Comment:
Давай добавим чуть больше ясности в названия
Мне пришлось подумать, что бы понять для чего это используется
```suggestion
    case develop
    case production
```